metabase/metabase
# Add `:if` as an alias to `:case` in the expression editor

Epic https://github.com/metabase/metabase/issues/50698
Depends on https://github.com/metabase/metabase/pull/51069

How to verify:
- New -> Question -> Products
- Custom column -> `if([Price] > 50, "Large", "Small")`
- Filter -> Custom expression -> `if([Price] > 50, "Large", "Small") = "Large"`
- Aggregation -> Custom expression -> `sum(if([Price] > 50, 1, 0))`
- Preview each step and make sure the results make sense

Note that there is a separate bug with expression matching in the filter widget atm https://github.com/metabase/metabase/pull/51089. E2E tests to be added separately.
